lib/diagnostic.js

```diff
@@ -15,7 +15,7 @@ const min = function (previous, current) {
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let total = array.reduce(sum);`
 const sum = function (previous, current) {
-
+  return Math.max(previous, current)
```

Comment:
This will return the largest element in an array, but you actually want to be adding them together. Something like `return previous + current`